assignments/callbacks.js

```diff
@@ -24,91 +24,58 @@ const items = ['Pencil', 'Notebook', 'yo-yo', 'Gum','yo-yo', 'Gum'];
 
 */
 
-
- 
-
 function getLength(arr, cb) {
-  return cb();
+  return cb(arr.length);
 }
-console.log(getLength(items, function () {
-  return items.length;
-}) );
- 
-  // getLength passes the length of the array into the callback.
+getLength(items, function(length) {
+  console.log(length);
+});
 
+// getLength passes the length of the array into the callback.
 
 function last(arr, cb) {
-  return cb();
+  return cb(arr.last);
 }
-console.log(last(items, function () {
-  return items.slice(-1)[0] ;
-}) );
-  // last passes the last item of the array into the callback.
-
-  function collectNumbers(x, y, cb){
-    return cb(x,y);
-  }
-
-  console.log(collectNumbers(5,6,function(x,y){return x + y}))
-  function addNumbers(x,y){
-    return x+y;
-  }
- // collectNumbers(5,6, addNumbers(5,6));
-
-function calculator(x,y){
-  return {
-    add: function() {
-      return x + y;
-    },
-    subtract: function() {
-      return x - y;
-    },
-    multiply: function() {
-      return x * y;
-    },
-    divide: function() {
-      return x / y;
-    }
-  }
-};
-
-console.log(calculator(7,3).multiply())
-
-   
-  // sumNums adds two numbers (x, y) and passes the result to the callback.
-
-function multiplyNumbers(x,y){
-  return x*y;
-};
-console.log(collectNumbers(5,6,multiplyNumbers));
-
-  // multiplyNums multiplies two numbers and passes the result to the callback.
+last(items, function(last) {
+  last = items.slice(-1)[0];
+  console.log(last);
+});
+
+// last passes the last item of the array into the callback.
+function sumNums(x, y, cb) {
+  return x + y;
+}
+console.log(sumNums(5, 6));
 
+// sumNums adds two numbers (x, y) and passes the result to the callback.
 
-function contains(item, list, cb){
- cb(list.includes(item));
- }
-console.log(items.includes('yo-yo'));
+function multiplyNumbers(x, y, cb) {
+  return x * y;
```

Comment:
Same here.  You're return the product of x and y but not passing it to the callback.